EasyPost.Net/Report.cs

```diff
@@ -66,16 +66,16 @@ public static Report Create(string type, Dictionary<string, object> parameters =
         /// </param>
         /// <param name="type">The type of report, e.g. "shipment", "tracker", "payment_log", etc.</param>
         /// <returns>Instance of EasyPost.ScanForm.</returns>
```

Comment:
update the comment

Reply:
Resolved in [73a3961](https://github.com/EasyPost/easypost-csharp/pull/223/commits/73a39618c0e40963cd3dd1b750ef53a13e56f3d1)

Reply:
I don't see the change in this line